solvers/python_pgd.py

```diff
@@ -6,19 +6,19 @@
 
 
 class Solver(BaseSolver):
-    name = 'Python-PGD'  # proximal gradient, optionally accelerated
+    name = "Python-PGD"  # proximal gradient, optionally accelerated
```

Comment:
can you revert these changes ? they are not needed, likely to complicate other PRs, and introduce noise

Reply:
sure. that was a ctrl + S in vscode that black-ify the file. Sorry about it.